golem-test-framework/src/components/worker_executor/docker.rs

```diff
@@ -48,25 +48,6 @@ impl DockerWorkerExecutor {
         worker_service: Arc<dyn WorkerService + Send + Sync + 'static>,
         verbosity: Level,
         shared_client: bool,
-    ) -> Self {
```

Comment:
did you check how cloud breaks without these? AFAIR these were introduced for that